newui/content/index.html

```diff
@@ -0,0 +1,233 @@
+    }
+    .greedy-nav button {
+      position: absolute;
+      top: 0;
+      right: 0;
+      height: 100%;
+    }
+  </style>
+<![endif]-->
+
+
+
+<link rel="icon" type="image/x-icon" href="/assets/images/favicon.ico">
+<link rel="stylesheet" href="/assets/css/font-awesome.min.css">
+
+  </head>
+
+  <body class="layout--home">
+    <!--[if lt IE 9]>
+<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
+<![endif]-->
+
+    
+
+<div class="masthead">
+  <div class="masthead__inner-wrap" id="masthead__inner-wrap">
+    <div class="masthead__menu">
+      <nav id="site-nav" class="greedy-nav">
+        
+          <a class="site-logo" href="/"><img src="/assets/images/hudi.png" alt=""></a>
+        
+        <a class="site-title" href="/">
+          Apache Hudi
+          <span class="site-subtitle">0.5.1-SNAPSHOT(incubating)</span>
+        </a>
+        <ul class="visible-links"><li class="masthead__menu-item">
+              <a href="/docs/quick-start-guide.html" target="_self" >Documentation</a>
+            </li><li class="masthead__menu-item">
+              <a href="/community.html" target="_self" >Community</a>
+            </li><li class="masthead__menu-item">
+              <a href="/roadmap.html" target="_self" >Roadmap</a>
+            </li><li class="masthead__menu-item">
+              <a href="/activity.html" target="_self" >Activities</a>
+            </li><li class="masthead__menu-item">
+              <a href="https://cwiki.apache.org/confluence/display/HUDI/FAQ" target="_blank" >FAQ</a>
+            </li><li class="masthead__menu-item">
+              <a href="/releases.html" target="_self" >Releases</a>
+            </li></ul>
+        <button class="greedy-nav__toggle hidden" type="button">
+          <span class="visually-hidden">Toggle menu</span>
+          <div class="navicon"></div>
+        </button>
+        <ul class="hidden-links hidden"></ul>
+      </nav>
+    </div>
+  </div>
+</div>
+
+
+    <div class="page__hero--overlay" style="background-color: #079fca !important;">
+      <div class="wrapper" style="background-size: 480px;background-repeat: no-repeat; background-position: right center;  background-image: url('/assets/images/logo-big.png');">
+        <h1 id="page-title" class="page__title" itemprop="headline">
+          Welcome to Apache Hudi
+        </h1>
+
+        <p class="page__lead">Apache Hudi Stands for Hadoop Upserts and Incrementals to manage the storage of large analytical datasets on HDFS.<br /> <small><a href="https://github.com/apache/incubator-hudi/releases/tag/release-0.5.0-incubating" target="_blank">Latest release 0.5.0-incubating</a></small>
+</p>
+        <p>
+          <a href="/docs/quick-start-guide.html" class="btn btn--light-outline btn--large"><i class="fa fa-paper-plane"></i> Get Starting</a>
```

Comment:
could not trace the source of this .. but can we make it "Get Started" instead of "Get Starting"

Reply:
Done.